Makefile

```diff
@@ -23,4 +34,20 @@ parsers/tree-sitter-tsx.wasm: node_modules/tree-sitter-typescript/tsx/package.js
 parsers/tree-sitter-c-sharp.wasm: node_modules/tree-sitter-c-sharp/package.json
 	mkdir -p $(dir $@)
 	npx tree-sitter build-wasm $(dir $^)
-	mv tree-sitter-c_sharp.wasm $@
\ No newline at end of file
+	mv tree-sitter-c_sharp.wasm $@
+
+
+# Build web-tree-sitter
+
+$(addprefix vendor/web-tree-sitter/,$(WEB_TREE_SITTER_FILES)):
```

Comment:
We have no dependency on the version number, from what I can tell, so how do we trigger this one to recompile?

Reply:
Good point, I should work the version into the path